src/components/questions/ImageUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  ToastAndroid,
+  Alert,
+  PermissionsAndroid,
+} from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+const ImageUpload = () => {
+  const [image, SetImage] = React.useState('');
+
+  // Show toast alert messages
+  const setToastMsg = (msg: any) => {
+    ToastAndroid.showWithGravity(msg, ToastAndroid.SHORT, ToastAndroid.CENTER);
+  };
+
+  const uploadImage = () => {
+    launchImageLibrary({ mediaType: 'photo', quality: 1, includeBase64: true }, (response) => {
+      if (response.didCancel) {
+        setToastMsg('geen foto gekozen');
+      } else if (response.errorCode == 'permission') {
+        setToastMsg('permissie afgewezen');
+      } else if (response.errorCode == 'others') {
+        setToastMsg(response.errorMessage);
```

Comment:
Willen we de gehele message laten zien? Ik zou meer iets doen in de vorm van `Er is een onbekende fout opgetreden.` en dat je hem wel logt. Wellicht een idee om als extra feature ook de logs van de app naar het dashboard te sturen zodat deze ook ergens opgeslagen staan buiten de app.

Reply:
we kunnen de extra feature nog later naar bekijken om het als log mee te kunnen geven aan dashboard, sinds camera en spraak opname belangrijk is voor de applicatie maak ik hiervoor een aparte utility file sinds het voor meerdere plekken toepasbaar zal worden voor de app zoals jaap benoemd heeft. Wel slim om later te kunnen loggen.